packages/tokens/src/tokens/breakpoint.ts

```diff
@@ -0,0 +1,7 @@
+export const breakpoint = {
+  sm: "600px",
+  md: "768px",
+  lg: "1024px",
+  xl: "1280px",
+  xxl: "1920px",
```

Comment:
Would it make more sense to make these something like "phone" "tablet" "desktop" or the like?

Reply:
Plus one here to the sizes you have here and to using `2xl` over `xxl`!

Reply:
Cool this works!